otlp/scripts/generate-modules.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if ! command -v protoc >/dev/null 2>&1; then
+  printf '%s\n' 'protoc is not available' 1>&2
+  exit 1
+fi
+
+if ! command -v proto-lens-protoc >/dev/null 2>&1; then
+  printf '%s\n' 'proto-lens-protoc is not available' 1>&2
+  exit 1
+fi
+
+PROTO_LENS=$(command -v proto-lens-protoc)
+
+OTLP_VERSION="$1"
```

Comment:
I think we should write this down in a file. Could just be a constant in this script.
1. That makes it easy to see what version we support without having to check the git history.
2. It lets us run a CI check to assert that the generated files haven't changed from what we would get if we re-generated them, which guards us against accidental modification (although that probably won't happen, and I don't think we should block this PR on having such a check).

Reply:
I guess we could even write it into a Haskell module and export it as a value from the library, although I'm not sure if there's actually any use for that!

Reply:
I created the `OTLP_VERSION` file.